string class_name
geometry_msgs/Pose pose
float32 width
int8 caution_level
float32 x1
float32 y1
float32 x2
float32 y2